Analysis/Tasks/PWGHF/taskLc.cxx

```diff
@@ -8,14 +8,14 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
 
-/// \file taskDPlus.cxx
-/// \brief D± analysis task
+/// \file taskLc.cxx
+/// \brief Lc± analysis task
```

Comment:
Add your name @DelloStritto 👍

Reply:
I would also consider renaming this LcpKpi in the future, soon we will also have the pK0s